.ci/jenkins_job_build.sh

```diff
@@ -45,12 +45,12 @@ export PATH=${GOPATH}/bin:/usr/local/go/bin:/usr/sbin:/sbin:${PATH}
 kata_repo_dir="${GOPATH}/src/${kata_repo}"
 tests_repo_dir="${GOPATH}/src/${tests_repo}"
 
-arch=$("${tests_repo_dir}/.ci/kata-arch.sh")
-
 # Get the tests repository
 mkdir -p $(dirname "${tests_repo_dir}")
 [ -d "${tests_repo_dir}" ] || git clone "https://${tests_repo}.git" "${tests_repo_dir}"
 
+arch=$("${tests_repo_dir}/.ci/kata-arch.sh)
```

Comment:
I suspect the thing that is different for this ARM job is maybe something in the Jenkins job setup that is different/missing between that and x86. Having said that, this change lgtm - if we are pulling the tests repo anyway just after we do the arch test, then moving the test down I think is fine.